css/cart.css

```diff
@@ -0,0 +1,14 @@
+@import "../common.css";
+
+.cart-container{
```

Comment:
Suggestion: Use space before { it looks good, use beautify extension in vs code

Reply:
Sure. I will apply it.